be/src/storage/lake/lake_persistent_index.cpp

```diff
@@ -408,6 +421,10 @@ Status LakePersistentIndex::load_from_lake_tablet(TabletManager* tablet_mgr, con
     auto rowsets = Rowset::get_rowsets(tablet_mgr, metadata);
     // Rowset whose version is between max_sstable_version and base_version should be recovered.
     for (auto& rowset : rowsets) {
+        TRACE_COUNTER_INCREMENT("total_rowsets", 1);
```

Comment:
add `load_` prefix to all these traces. E.g. `total_rowsets` -> `load_total_rowsets`.